apps/aechannel/src/aesc_fsm.hrl

```diff
@@ -227,8 +227,11 @@
                         | ?DEP_ERR
                         | ?WDRAW_ERR.
 
--opaque opts() :: #{ minimum_depth          => non_neg_integer() %% Defaulted for responder, not for initiator.
-                   , minimum_depth_strategy => txfee
+-type minimum_depth_factor()   :: non_neg_integer().
+-type minimum_depth_strategy() :: txfee.
```

Comment:
Should we add an instant strategy in which we only require only a micro block confirmation?

Reply:
Yes, we should add a `static` strategy where the factor equals the min depth and `0` means only a micro block confirmation is needed.